extensions/duplicates/README.md

```diff
@@ -6,9 +6,9 @@ dependency graph.
 [phylum-cli]: https://github.com/phylum-dev/cli
 
 This extension serves as a simple example and has its source code fully
-explained in our [extension example documentation].
+explained in our [extension example documentation][example].
 
-[extension example documentation]: https://docs.phylum.io/docs/extension_example
+[example]: https://docs.phylum.io/cli/extensions/extension_example
```

Comment:
Did this not work or why did you change the link ID here? That usually shouldn't cause a problem.

Reply:
It worked, but spilled over past the 80 character mark. The change was an attempt to keep all lines within that limit. It has been changed back.